src/tests/ftest/rebuild/mdtest.yaml

```diff
@@ -0,0 +1,57 @@
+hosts:
+  test_servers: 2
+  test_clients: 2
+
+timeout: 900
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      targets: 8
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      log_mask: INFO
+      storage: auto
+    1:
+      targets: 8
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      log_mask: INFO
+      storage: auto
+
+pool:
+  size: 95%
+  properties: rd_fac:1
+  rebuild_timeout: 720  # 12 minutes
+  rebuild_check_inverval: 30  # check every 30 seconds
+
+container:
+  type: POSIX
+  control_method: daos
+
+mpirun:
+  args: "--bind-to hwthread --map-by socket"
+
+mdtest:
+  client_processes:
+    np: 64
+  api: DFS
+  dfs_oclass: EC_2P1GX
```

Comment:
I thought we use G1 type of oclass [eg: EC_2P1G1]... Is it not?

Reply:
It depends on what we're testing. GX will use all targets, so can be useful for rebuild testing. And in this case, the ticket specifically requested EC_2P1GX
https://daosio.atlassian.net/browse/DAOS-15197